cfg/params.yaml

```diff
@@ -628,6 +628,17 @@
   default: 64 #TODO: revisit default value after perf testing.
   hide-flag: true
 
+- config-path: "write.chunk-transfer-timeout"
+  flag-name: "chunk-transfer-timeout"
```

Comment:
We are suffixing the unit for other flags e.g. "kernel-list-cache-ttl-secs" and create an int flag instead of duration. Maybe we should follow the same convention here.

Reply:
Done